tests/tracking/fluent/test_fluent_autolog.py

```diff
@@ -218,10 +218,16 @@ def test_autolog_obeys_disabled():
 
     mlflow.autolog(disable=False)
     mlflow.utils.import_hooks.notify_module_loaded(sklearn)
-    assert not get_autologging_config("sklearn", "disable", False)
+    assert not get_autologging_config("sklearn", "disable")
     mlflow.sklearn.autolog(disable=True)
     assert get_autologging_config("sklearn", "disable")
 
+    mlflow.autolog(disable_for_unsupported_versions=False)
+    mlflow.utils.import_hooks.notify_module_loaded(sklearn)
+    assert not get_autologging_config("sklearn", "disable_for_unsupported_versions")
+    mlflow.sklearn.autolog(disable_for_unsupported_versions=True)
+    assert get_autologging_config("sklearn", "disable_for_unsupported_versions")
+
```

Comment:
Cool! Can we also add the `disable_for_unsupported_versions` argument to the `test_universal_autolog_calls_specific_autologs_correctly` test case to ensure that it's propagated for all supported autologging frameworks?